portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java

```diff
@@ -137,21 +72,25 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         catch(EntityNotFoundException e){}
     }
 
-    
 
+
+    Entity input = addProperties(currentDate, request, userNickname);
+    datastore.put(input);
+
+  }
+
+  public static Entity addProperties(String currentDate, HttpServletRequest request, String userNickname){
     //Adds day, value, and question to arraylist
     List<String> properties = new ArrayList<>();
-    properties.add(storedDate);
+    properties.add(currentDate);
```

Comment:
It may help to store this as a timestamp instead of a string

Reply:
The only thing is that a timestamp saves the time and the date, when I only need the date. Also, converting timestamp to a date object in javascript would also save the time and then I have to format it so it displays as just the date and then I would have to put it back into my array. I just prefer it how it is, at least for right now.